framework/src/st_things/things_stack/src/stack/things_stack.c

```diff
@@ -189,8 +189,12 @@ static void *__attribute__((optimize("O0"))) t_things_wifi_join_loop(void *args)
 	return NULL;
 }
 
-void things_wifi_sta_connected(void)
+void things_wifi_sta_connected(wifi_manager_result_e res)
 {
+	if (res == WIFI_MANAGER_FAIL) {
```

Comment:
Is the WiFi manager mandantory for st_things? You didn't check a conditional.

Reply:
WiFi manager is mandatory for st_things.